packages/template/src/lib/stack-app.ts

```diff
@@ -2676,6 +2680,12 @@ class _StackAdminAppImpl<HasTokenStore extends boolean, ProjectId extends string
     return crud.token;
   }
   // END_PLATFORM react-like
+  // IF_PLATFORM react-like
+  useSentEmails(): InternalEmailsCrud["Admin"]["List"] {
+    const crud = useAsyncCache(this._sentEmailsCache, [], "useSentEmails()");
+    return crud;
```

Comment:
you should never return a crud type from the client library; those use `snake_case`, whereas the client library should only return `camelCase` objects

(see how the other objects do this)